### DISCUSSION TITLE
My Espressif’s Thread Border Router won't connect to my WiFi network

### DISCUSSION THREAD

**Discussion by mforbiz** [openthread/openthread#10047] [Q&A] [answered]
Hi.
I've configured my Espressif’s Thread Border Router board following the offical guide:
[https://openthread.io/guides/border-router/espressif-esp32](url)
and everything seems to have gone well.

Before to flash esp32s3 ,and using
`idf.py menuconfig`
I've configured by means:
`Example Connection Configuration > connect using Wi-Fi interface` 
Wifi SSID and password to connect to my WiFi nework.

Neverthless the board won't connect to my WiFi newtork and I can't access it in anyway via GUI.
Any suggestions?

Thanks in advance

**Comment by chshu** [comment]
@mforbiz There could be various reasons that may cause the Wi-Fi connect failure, could you double check the following:

- the Wi-Fi AP availability, are the Wi-Fi SSID and password correct?
- Does the Wi-Fi AP support 2.4G or 5G? The esp32s3 supports 2.4G only

If these check don't help, please collect the device log and the Wi-Fi AP information, and file an issue here: https://github.com/espressif/esp-thread-br/issues, we will get back to you soon.

**Reply by mforbiz** [reply, marked as answer]
Hi chshu.
Thanks for you reply.
I've solved the issue using the ethernet interface (_daughter board)_ and this settings (following https://docs.espressif.com/projects/esp-thread-br/en/latest/dev-guide/build_and_run.html) :

![image](https://github.com/openthread/openthread/assets/105542757/e0b4e349-6550-4d28-b290-825863a8c90d)

![image](https://github.com/openthread/openthread/assets/105542757/22373302-07f7-4ab6-a5a1-b946c4698d6a)
Now all seems to work fine.

![image](https://github.com/openthread/openthread/assets/105542757/bc1f1492-a9c6-472d-97c6-5358d0c0a492)
